prophic3/array_axiom_enumerator.h

```diff
@@ -55,17 +56,58 @@ class ArrayAxiomEnumerator {
       }
     }
 
+    // remove witnesses from curr_indices_no_witnesses_
```

Comment:
didn't we agree that we need to keep witness indices?
I think we were giving the wrong answer to a chc benchmark (giving unsafe instead of safe).

Reply:
Yeah that’s right, but we’re still including witnesses. I only took them out from the enumerators grammar search. Just a heuristic, but it seems strange to me that we’d ever need to prophesize something like `witness1 + x`. Because, the witnesses are just arbitrary indices, there should be no reason to do arithmetic on them.

Reply:
i see